index.bs

```diff
@@ -2409,31 +2431,29 @@ Note: Extensions should aim to define authenticator arguments that are as small
 ## <dfn>Client extension processing</dfn> ## {#sctn-client-extension-processing}
 
 Extensions may define additional processing requirements on the client platform during the creation of credentials or the
-generation of an assertion. In order for the [=[RP]=] to verify the processing took place, or if the processing has a result
-value that the [RP] needs to be aware of, the extension specifies a extension key-value pair to be included in the [=client data=] extensions field,
-which is returned to the [RP] via {{AuthenticatorResponse/clientDataJSON}}.
-
-If any extension processed by a client defines such a
-value, the client includes a dictionary in its [=client data=] with the key {{CollectedClientData/extensions}}. For each such
+generation of an assertion.
+The [=client extension input=] for the extension is used an input to this client processing.
+Supported [=client extensions=] are recorded as a dictionary in the [=client data=] with the key {{CollectedClientData/extensions}}.
+For each such
 extension, the client adds an entry to this dictionary with the [=extension identifier=] as the key, and the extension's
 [=client extension input=] as the value.
-The [=client extension input=] value may be any value that can be encoded using JSON.
 
-Likewise, the extension output is represented in the [=client data=] as a CBOR map
+Likewise, the [=client extension outputs=] are represented as a dictionary in the {{ScopedCredential/clientExtensionResults}}
 with [=extension identifiers=] as keys, and the <dfn>client extension output</dfn> value of each extension as the value.
+Like the [=client extension input=], the [=client extension ouput=] is a value that can be encoded in JSON.
```

Comment:
Typo: "ouput".

You can check https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/selfissued/webauthn/mbj-extensions-connective-tissue/index.bs&output=err for this sort of error.